rke/k8s_service_options.go

```diff
@@ -15,6 +15,14 @@ const (
 
 func loadK8sVersionServiceOptions() map[string]v3.KubernetesServicesOptions {
 	return map[string]v3.KubernetesServicesOptions{
+		"v1.24": {
+			Etcd:           getETCDOptions122v352(),
```

Comment:
we'd use `getETCDOptions122()` here, `122v352()` was to temporarily enable `experimental-initial-corrupt-check` for versions using etcd v3.5.0-v3.5.2 for a security issue. Now that we're using latest etcd, we can continue using the older option

Reply:
switch back to `getETCDOptions122()`